src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] `README.md` file: Kindly make sure you update your some of your README sections with appropriate content information to make your documentation more professional. Such as:
```
## Setup
Clone this repository to your desired folder:

git clone https://github.com/muhammedb8826/Calculator.git

cd Calculator
```
```
## Live Demo
Coming soon!
Link to github repository.

------------> [To-Do list app](https://github.com/muhammedb8826/Calculator)
```
| Setup | Live Demo |
| -------------| --------------- |
| ![Screenshot from 2023-03-22 22-42-11](https://user-images.githubusercontent.com/83042742/227087903-2b7bf718-e016-46a9-ac44-3220bd2d0144.png) | ![Screenshot from 2023-03-22 22-47-47](https://user-images.githubusercontent.com/83042742/227087791-d1bddd29-442e-41b5-be80-136820bc980a.png) |